README.md

```diff
@@ -43,6 +43,46 @@ MG serve to capture the **definitions** of terms introduced in the MIPs and MDs.
 
 
 ## Files and numbering
-Each MIP, MD or MG is stored in a separate subdirectory with the a name `mip-<number>`, `md-<number>` or `mg-<number>`. The subdirectory contains a README.md file that describes the MIP, MD, or MG. All assets related to the MIP, MD or MG are stored in the same subdirectory.
+Each MIP, MD or MG is stored in a separate subdirectory with the a name `mip-<number>`, `md-<number>` or `mg-<number>`. The subdirectory contains a `README.md` file that describes the MIP, MD, or MG. All assets related to the MIP, MD or MG are stored in the same subdirectory.
 
-MIPs, MDs and MGs are assigned their PR number as soon as they are drafted. MDs that do not introduce a new MIP are also accepted. Thus, there will be gaps in the MIP number sequence. These gaps will also emerge when MIPs are deprecated or rejected.
+MIPs start as Drafts. MIPs obtain a number during the review process. The number is the highest number in the MIP and MD `overview.md` file plus 1.
+
+PRs that don't introduce a new MIP are also accepted. 
+- MIPs can be updated. PRs that update a MIP should state so in the PR title via `[Update] .... `. 
+
+There may be gaps in the numbering, as MIPs get rejected and removed.
+
+
+
+## Status Terms
+An MIP is proposed through a PR. Each MIP-introducing PR should have a status in the name in the form `[Status] ...`.
+
+An MIP should at all times have one of the following statuses:
+- **Draft** - (set by author) An MIP that is open for consideration. (It does not yet hold an MIP number)
+- **Review** - (set by author) The MIP is under peer review. The MIP should receive an **MIP number**, according to the rules described in the [Files and numbering](#files-and-numbering) section.
+
+>[!Note]
+> In case the editors are not available for an unacceptable long period of time, a reviewer should assume the role of the editor interim. 
+
```

Comment:
I might note in this very README that a big motivation for the role of the editor is the native readability of this format. We won't likely have something that re-renders the content herein for a while, so for people new to the format--particularly external users--the quality the editor ensures is important.

Reply:
ok done at the bottom section